docusaurus-utils/scrapper/fetch-content.mjs

```diff
@@ -47,12 +50,31 @@ export async function fetchDirectoryContents(dirUrl, basePath, prefixToRemove) {
       const filePath = path.join(basePath, relativePath)
```

Comment:
All the files get parsed later on which is most of the scrapper codebase. The docosaurus .md syntax is different to regular .md

Reply:
We also don't want any .md files to be part of this repo and keep index as the source of truth

Reply:
Why would we keep it in the index? I see in the PR you already added the download paths to `.gitignore`, so would they be added to the repo?

Reply:
.gitignore is there just a safety mechanism if someone is working locally to not accidentally commit these things, they aren't supposed to be part of the repo but just be generated for the purpose of the build.

It was a request from Vac to keep index and website repos separate

Reply:
They wouldn't ever be added to the repo, they're generated only for the purpose of the build. They're kept in root because docosaurus when building, creates router structure based on the file system at the moment of building

/waku/standards/...
is different from
/some-temp-folder/waku/standards/...